### DISCUSSION TITLE
Extension activation failed: "Failed to get copilot token"

### DISCUSSION THREAD

**Discussion by Dgarzona16** [community/community#7714] [Copilot Conversations] [answered]
can you help me? pls

**Comment by davecheney** [comment, marked as answer]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by Dgarzona16** [reply]
I do not use vpn, I could disable the firewall and try, the only thing I did recently was to change OS

**Reply by Dgarzona16** [reply]
I already disable the firewall and I still have the problem, in the output I get some numbers that I think are the tokens and if they are different

**Reply by Edi-Fatemi** [reply]
i have same problem after changing windows.
